api/v1alpha4/machineset_webhook_test.go

```diff
@@ -36,7 +35,6 @@ func TestMachineSetDefault(t *testing.T) {
 	md.Default()
 
 	g.Expect(md.Labels[ClusterLabelName]).To(Equal(md.Spec.ClusterName))
-	g.Expect(md.Spec.Replicas).To(Equal(pointer.Int32Ptr(1)))
```

Comment:
The test should probably be moved elsewhere

Reply:
Is there a way to test validation/default kubebuilder rules? I haven't seen any place where projects do that. We currently have `make verify` that can catch this.
Perhaps this could be included in the e2e tests but I don't see a clear place to include it.

Reply:
Creating the resource through a test env environment should do it

Reply:
@vincepri Yeah that makes sense.
@alexander-demichev We can either include a similar `suite_test.go` that we have in the `controllers` package in the `v1alpha4` or just include a `schema_test.go` in the `controllers` package perhaps.

WDYT?

Reply:
@wfernandes I think it's easier to include a schema_test.go in controllers package.